db/data/seedDatabase.js

```diff
@@ -1,40 +1,103 @@
-const makeFakeRestaurant = require('./fakeDataGenerator.js');
+require('dotenv').config();
 const { MongoClient } = require('mongodb');
+const cluster = require('cluster');
+const numCPUs = require('os').cpus().length;
 
-const batchSize = parseInt(process.env.BATCH_SIZE, 10) || 10000;
-const numBatches = parseInt(process.env.NUM_BATCHES, 10) || 1000;
+const makeFakeRestaurant = require('./fakeDataGenerator.js');
 
-const url = 'mongodb://localhost:27017';
+const batchSize = parseInt(process.env.BATCH_SIZE, 10) || 10000;
+const numRecords = parseInt(process.env.NUM_RECORDS, 10) || 10000000;
 
-MongoClient.connect(url, (err, client) => {
+const connectToDb = async () => {
+  const url = 'mongodb://localhost:27017';
+  const client = await MongoClient.connect(url);
   const restaurants = client.db('businessinfo').collection('restaurants');
+  return { client, restaurants };
+};
 
-  async function seedDatabase() {
-    await client.db('businessinfo').dropDatabase();
-    console.log('Dropped database');
+const indexDb = async (restaurants) => {
```

Comment:
minor style-guide type stuff but some of these function declarations could probably use a semicolon at the end! Seeing this here, and on lines 36, 35, 41, etc.

Reply:
thanks. yeah it appears my linter turned off. thanks for catching those. changed!